testExe/dune

```diff
@@ -0,0 +1,5 @@
+(executable
+ (name RunReasonTests)
+ (public_name RunReasonTests.exe)
+ (package reason)
```

Comment:
Along with my other comment, you would make it be part of the `reason-test` "package".

Reply:
when i make it part of the reason-test package and run esy x test i get the following error: 

```
error: unable to resolve command: RunReasonTests.exe

esy: exiting due to errors above
```

Reply:
@jordwalke do you know what to do, i'll also have a look on other projects

Reply:
looking at this at the moment https://github.com/facebookexperimental/reason-native/blob/de93a483355d9133be42ff25ace7a3afdcf03974/tests/dune

Reply:
Yeah, I think I know why it does this. `esy x cmd` simulates your package environment from the perspective of someone that installed it / depended on it. Therefore it doesn't run the `buildDev` command - but the regular build command.

I think I just realized some good news though: You don't need to make a separate `buildDev` command - you can just use the regular `build` command. The reason why, is that we won't actually ship this `esy.json` file - no we have smaller separate `.json` files for each package `reason.json`/`rtop.json` etc. So no worries about the separate buildDev command - that should clear up your challenge.